db/migrate/20220827125322_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Reference name should be an author. Kindly fix it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)